hydra-node/src/Hydra/Options.hs

```diff
@@ -124,6 +126,9 @@ publishOptionsParser =
     <*> nodeSocketParser
     <*> cardanoSigningKeyFileParser
 
+deriving instance ToJSON IP
+deriving instance FromJSON IP
```

Comment:
Can you move these please to `Hydra.Network`? We do re-export `IP` there, so the instances could fit there nicely as well.